.gitignore

```diff
@@ -3,6 +3,8 @@
 
 .bsp/
 .idea/
+.sbt/
+null/
```

Comment:
Hmm where's that directory coming from? 🤔

Reply:
Judging by internal folder structure, it was created by `sbt-projectmatrix` plugin